serverless/aws/protomaps-template.yaml

```diff
@@ -1,36 +1,39 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: CloudFormation template to create a protomaps infraestructure to serve tiles.
+Description: Serve Z/X/Y tiles through CloudFront + Lambda from an existing S3 bucket.
 Parameters:
   BucketName:
-    Description: 'The name of the S3 bucket where you will store pmtiles files to be served (must be globally unique)'
+    Description: 'Name of an existing S3 bucket with .pmtiles tilesets. Should be in the same region as your CloudFormation stack.'
     Type: String
+    MinLength: 1
 
-  CodeBucketName:
```

Comment:
For now let's use a single bucket that contains both `lambda_function.zip` and the PMTiles, until we have a automated way to inline the code.

I believe inlining is a better solution because otherwise code updates to `lambda_function.zip` require a key change or versioning, since CloudFormation does not detect changes to S3 assets.